src/JsonPatch/JsonPatchDocument.cs

```diff
@@ -1,13 +1,8 @@
-﻿using System;
+﻿using JsonPatch.Formatting;
+using JsonPatch.Paths.Resolvers;
+using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Reflection;
-using System.Text;
-using System.Threading.Tasks;
-using JsonPatch;
-using JsonPatch.Paths;
-using JsonPatch.Paths.Resolvers;
-using JsonPatch.Formatting;
```

Comment:
Noticed in this file I missed cleaning up other unused references in my last pull request, this is a small cleanup item I am adding in.